lib/android.js

```diff
@@ -2316,6 +2317,7 @@ class ArtMethodMangler {
     // Remove kAccFastInterpreterToInterpreterInvoke and kAccSkipAccessChecks to disable use_fast_path
     // in interpreter_common.h
     let hookedMethodRemovedFlags = kAccFastInterpreterToInterpreterInvoke;
+    hookedMethodRemovedFlags |= kAccSingleImplementation;
```

Comment:
```suggestion
    let hookedMethodRemovedFlags = kAccFastInterpreterToInterpreterInvoke | kAccSingleImplementation;
```

Reply:
(Would be good to run `node_modules/.bin/semistandard` after this change – not sure if I exceeded the line length in this suggestion here.)